<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UIAdministrateur</class>
 <widget class="QMainWindow" name="UIAdministrateur">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1204</width>
    <height>608</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Interface administrateur</string>
  </property>
  <property name="windowIcon">
   <iconset resource="src.qrc">
    <normaloff>:/image/responsable.png</normaloff>:/image/responsable.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#centralwidget{
 border-image:url(:/image/bg.jpg);

}

#label{

color: white;

  font-family: Arial;

}

#label_2,#label_3,#label_5,#label_6,#label_8,#label_7,#label_9,#label_10,#label_11,#label_4,#labelRole_2,#labelPassword_2,#label_12{

color: white;
 
  font-family: Arial;

}

#labelId{

color: white;
 
  font-family: Arial;

}

#labelConfirmPassword_2{

color: white;
 
  font-family: Arial;

}

#labelPassword{

color: white;
 
  font-family: Arial;

}


#labelUsername{

color: white;
 
  font-family: Arial;

}

#radioButtonCreer{

color: white;
 
  font-family: Arial;

}

#radioButtonModifier{

color: white;
 
  font-family: Arial;

}

#labelUtilisateurConnecteTitre{

color: white;
 
  font-family: Arial;

}

#labelUtilisateurConnecte{

color: white;
 
  font-family: Arial;
}

#labelRole{

color: white;
 
  font-family: Arial;
}

#labelRechercher{

color: white;
 
  font-family: Arial;
}

 QLineEdit {
      border: 2px solid gray;
      
      padding: 0 8px;
      
      selection-background-color: darkgray;
  }


QPushButton
{
  color: white;
  background-color: #27a9e3;
  border-width: 0px;
  border-radius: 3px;
}

QPushButton:hover { background-color: #66c011; }





#comboBoxRole{

 color: white;
  background-color: #27a9e3;
  border-width: 0px;
  border-radius: 3px;
}
#comboBoxRole:hover { background-color: #66c011; }

#comboBoxRechercherPar{

 color: white;
  background-color: #27a9e3;
  border-width: 0px;
  border-radius: 3px;
}



QGroupBox{

color: white;

}



</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>490</y>
      <width>1181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>510</y>
      <width>1191</width>
      <height>37</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonValider">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>35</height>
        </size>
       </property>
       <property name="text">
        <string>Valider</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonLister">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>35</height>
        </size>
       </property>
       <property name="text">
        <string>Lister</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonSupprimer">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>35</height>
        </size>
       </property>
       <property name="text">
        <string>Supprimer</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonEffacer">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>35</height>
        </size>
       </property>
       <property name="text">
        <string>Effacer</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonExit">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>35</height>
        </size>
       </property>
       <property name="text">
        <string>Exit</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>60</y>
      <width>641</width>
      <height>421</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>10</y>
      <width>1181</width>
      <height>47</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="pushButtonProfil">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>35</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Consulter mon profil ...</string>
       </property>
       <property name="text">
        <string>Profile</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelRechercher">
       <property name="text">
        <string>Rechercher un utilisateur à partir de son</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxRechercherPar">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <item>
        <property name="text">
         <string>Id</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Username</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditRechercher">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonRechercher">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>35</height>
        </size>
       </property>
       <property name="text">
        <string>Rechercher</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>560</y>
      <width>221</width>
      <height>32</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="labelUtilisateurConnecteTitre">
       <property name="text">
        <string>Utilisateur Connecté :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelUtilisateurConnecte">
       <property name="text">
        <string>&lt;dynamic&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>531</width>
      <height>421</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="title">
     <string> ENREGISTREMENT D'UN  USER</string>
    </property>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>501</width>
       <height>181</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="title">
      <string>INFORMATION SUR USER</string>
     </property>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Noms :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Numero CNI : </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_CNI">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>100</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditUsername">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>60</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>60</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Prenoms :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditprenome">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>60</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Date Naiss :</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEdit_Naiss">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>140</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>100</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Lieu Naiss :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_lieu">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>100</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>140</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Sexe :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>47</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Id :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditId">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>20</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Matricule">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>20</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>20</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Matricule :</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxRole_Sexe">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>140</y>
        <width>151</width>
        <height>30</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">#comboBoxRole_Sexe
{
  color: white;
  background-color: #27a9e3;
  border-width: 0px;
  border-radius: 3px;
}

#comboBoxRole_Sexe:hover { background-color: #66c011; }

</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <item>
       <property name="text">
        <string>Homme</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Femme</string>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>381</width>
       <height>141</height>
      </rect>
     </property>
     <property name="title">
      <string>INFORMATION PROFESSOINNELLES</string>
     </property>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Tel :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Tel">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>113</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxRole">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>70</y>
        <width>111</width>
        <height>30</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">#comboBoxRole_2
{
  color: white;
  background-color: #27a9e3;
  border-width: 0px;
  border-radius: 3px;
}

#comboBoxRole_2:hover { background-color: #66c011; }

</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <item>
       <property name="text">
        <string>ADMINISTRATEUR</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>RESPONSABLE</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>FORMATEUR</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ETUDIANT</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="labelRole_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>27</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Rôle :</string>
      </property>
     </widget>
     <widget class="QWidget" name="formLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>20</y>
        <width>211</width>
        <height>104</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Login :</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEditLogin">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="labelPassword_2">
         <property name="text">
          <string>Password :</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEditPassword">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="lineEditConfirmPassword">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="labelConfirmPassword_2">
         <property name="text">
          <string>Confirm password :</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>230</y>
       <width>111</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#label_10{ border: 1px solid;
      border-image:url(:/image/u.jpg);

}</string>
     </property>
     <property name="text">
      <string>PHOTO</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>330</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Inserer la photo</string>
     </property>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>370</y>
       <width>250</width>
       <height>39</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="radioButtonCreer">
        <property name="text">
         <string>Creer</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButtonModifier">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Modifier</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="src.qrc"/>
 </resources>
 <connections/>
</ui>
